runtime/wasm/effect.wat

```diff
@@ -123,8 +123,6 @@
             (field $suspender externref)
             (field $next (ref null $fiber)))))
 
-   (type $continuation (struct (mut eqref)))
-
```

Comment:
I suppose you switched from a struct to an array in order to be compatible with potential future runtime primitives that will look at the the tag of continuations?

Reply:
The issue is that the code in `effect.ml` now performs an affectation (`"%setfield1`), which only makes sense if continuations have type `(ref $block)`.
https://github.com/lpw25/ocaml/blob/3437f2e52301315c74867ef2207584c54097981b/stdlib/effect.ml#L97

Reply:
Ah, I see.